backend/model/user.model.js

```diff
@@ -19,14 +19,17 @@ const userSchema = new Schema({
 
 userSchema.pre('save', function (next){
     const user = this
-    // console.log(this.isModified('password'))
-    // console.log(this)
     if(this.isModified('password')){
         bcrypt.hash(user.password, saltRounds).then((hashedPassword) => {
-            // console.log(hashedPassword)
             user.password = hashedPassword
             next()
         })
+    }if(this.isModified('firstName')){

```

Comment:
i dont think this is necessary here! line 27-32 as the user is already being updated in user.route